web/profiles/custom/os2loop/themes/os2loop_theme/templates/content-entities/paragraph--os2loop-documents-step.html.twig

```diff
@@ -51,24 +51,28 @@
 
 {% set has_title = content.os2loop_documents_step_title['#title'] is not null %}
 
+{% set expanded = app.request.query.get('expanded') %}
+{% set class_collapsed = not expanded ? 'collapsed' %}
+{% set class_collapse = not expanded ? 'collapse' %}
+
 {% block paragraph %}
   {% block content %}
     <li {{ attributes.addClass(classes) }}>
       {# A hack to check if we're rendering content for pdf generation #}
       {% if 'print/pdf' in url('<current>')['#markup'] %}
-        <span class="collapsed step--collapse-toggle">
+        <span class="{{ class_collapsed }} step--collapse-toggle">
           <span class="bold">
             {{ has_title ? content.os2loop_documents_step_title : content.os2loop_documents_step_text }}
           </span>
         </span>
       {% else %}
-        <a class="collapsed step--collapse-toggle" type="button" data-toggle="collapse" data-target="#paragraph-id-{{ paragraph.id() }}">
+        <a class="{{ class_collapsed }} step--collapse-toggle" type="button" data-toggle="{{ class_collapse }}" data-target="#paragraph-id-{{ paragraph.id() }}">
```

Comment:
why not button?

Reply:
Because I didn't write the code. But it makes sense to change it.

Reply:
The `a` element contains the step header (title) and a `button` element may not be appropriate for this use.